keyboards/sofle_choc/sofle_choc.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2022 Brian Low
```

Comment:
This file can be deleted.

Reply:
Then I get:

```
Compiling: .build/obj_sofle_choc/src/default_keyboard.c                                            In file included from .build/obj_sofle_choc/src/default_keyboard.c:26:
./.build/obj_sofle_choc/src/default_keyboard.h:43:10: fatal error: sofle_choc.h: No such file or directory
   43 | #include "sofle_choc.h"
      |          ^~~~~~~~~~~~~~
compilation terminated.
```

Reply:
Nevermind, this got fixed after forcefully regenerating `default_keyboard.h`